src/engraving/layout/layoutsystem.cpp

```diff
@@ -1209,6 +1209,18 @@ void LayoutSystem::layoutSystemElements(const LayoutOptions& options, LayoutCont
             }
         }
     }
+
+    //-------------------------------------------------------------
+    // Harp pedal diagrams
```

Comment:
By putting this at the bottom of the function, harp pedal diagrams will be stacked 'outside' of everything else. For example, if they are above the staff, they will appear even above tempo markings and rehearsal marks. @oktophonie What would be the correct stacking order?

Reply:
@miiizen, as suggested by @oktophonie, let's place them for now "outside" ottavas but "inside" anything else. That would mean moving this block of code to just above https://github.com/musescore/MuseScore/blob/3ee3c7eaa517cfa0aca563fae476739f4ca2b894/src/engraving/layout/layoutsystem.cpp#L1090-L1094